src/utils/HTMLCollection.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Class provides interface for HTLMCollection. Used in getElementsByTagName etc.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection
+ */
+export default class HTMLCollection extends Array {
+    constructor(parent, selector) {
+        super();
+        this._parent = parent;
+        this._selector = selector;
+        this._fillChildren(this._parent);
+    }
+
+    _fillChildren(parent) {
```

Comment:
HtmlCollection is supposed to be a live collection: I suppose that this implementation doesn't work

test:

```js
const document = new Document();
const divCollection = document.body.getElementsByTagName('div');
document.body.appendChild(document.createElement('div'));
strictEqual(divCollection.length, 1);
```